### ISSUE TITLE
[V13] Remediate synthetic universe, valid-observation windows, criterion J, and Q4 audit

### ISSUE THREAD

**Issue by ta1k1-arakawa** [ta1k1-arakawa/stock-analyzer#33]
## Status

```text
TASK_STATUS=READY
REPO=ta1k1-arakawa/stock-analyzer
AUTHORITATIVE_BRANCH=v13-conditional-cross-sectional-short-horizon
EXPECTED_HEAD=e2d3a870428e7706e751915fc46ff468f42996cf
TASK=V13_SYNTHETIC_FEASIBILITY_METHOD_PRESERVING_REMEDIATION_3
EXECUTOR_CLASS=STRONG_CODEX_REQUIRED
GPT_FINAL_REVIEW_AUTHORITY=GPT-5.6_SOL
REMEDIATION_CLASS=METHODOLOGY_PRESERVING_IMPLEMENTATION_FIX
SCIENTIFIC_REMEDIATION_HARD_CAP=NOT_APPLICABLE
ENGINEERING_ESCALATION_THRESHOLD_MINUTES=240
CUMULATIVE_ACTIVE_IMPLEMENTATION_MINUTES_BEFORE_TASK=46
```

## Purpose

Remediate the remaining methodology-preserving implementation/proof defects from the GPT exact-SHA review of Issue #31 commit `3a75fd93cc51c122ea78a56f6a1699ac93d3c786`, under the human-approved Issue #32 policy amendment now GPT-reviewed PASS at `e2d3a870428e7706e751915fc46ff468f42996cf`.

This task does not change V13 scientific methodology and does not authorize real acquisition, real ticker selection, historical-price access, outcome-bearing model fit/backtest, forward paper, private/sealed access, or trading.

## Read first

- AGENTS.md
- AI_GITHUB_ISSUE_ORCHESTRATION_WORKFLOW.md
- AI_RESEARCH_EXECUTION_RULES.md
- AI_STOCK_ANALYZER_REVIEW_POLICY.md
- AI_RESEARCH_CHECKPOINT_WORKFLOW.md
- AI_RESEARCH_DEVELOPMENT_EFFICIENCY_WORKFLOW.md
- V13_IMPLEMENTATION_REMEDIATION_POLICY_AMENDMENT.md
- V13_CONDITIONAL_CROSS_SECTIONAL_SHORT_HORIZON_DESIGN_DRAFT.md
- V13_PRE_IMPLEMENTATION_DETERMINISM_AMENDMENT_DRAFT.md
- V13_DESIGN_FREEZE_APPROVAL.json
- V13_PRE_IMPLEMENTATION_AMENDMENT_FREEZE_APPROVAL.json
- V13_PRE_IMPLEMENTATION_CONTRACT_MATRIX.md
- PROJECT_STATE.md
- PROJECT_DECISION_LOG.md
- Issue #31 GPT BLOCK comment
- Issue #32 GPT PASS comment

## Findings to close

```text
HIGH_1=SYNTHETIC_UNIVERSE_CHAIN_NOT_BOUND_TO_FROZEN_SELECTION
HIGH_2=VALID_OBSERVATION_ROLLING_CONTRACT_VIOLATED
HIGH_3=CRITERION_J_THRESHOLD_MISMATCH
MEDIUM_1=Q4_AUDIT_IS_SELF_ASSIGNED_NOT_INDEPENDENTLY_DERIVED
LOW_1=USER_SPECIFIC_ABSOLUTE_PATH_COMMITTED_SAFE_DEFERRED_APPEND_ONLY_HISTORY
```

LOW_1 is historical append-only log content and is safe to defer. Do not rewrite historical decision-log entries merely to erase it; do not add any new user-specific absolute path.

## Allowed files only

- `src/v13_feasibility.py`
- `src/v13_synthetic_fixture.py`
- `tests/test_v13_feasibility.py`
- `V13_PRE_IMPLEMENTATION_CONTRACT_MATRIX.md`
- `PROJECT_STATE.md`
- `PROJECT_DECISION_LOG.md`

Exactly these six files. Do not edit the CLI unless a newly discovered mechanical necessity makes the scope insufficient; if so STOP with `CHATGPT_DECISION_REQUIRED` rather than broadening scope.

## Environment

Follow the reviewed generated-worktree existing-venv rule. Use an already-existing project `.venv\Scripts\python.exe` that imports numpy/pandas/lightgbm/sklearn/scipy/pytest. No package or environment mutation. No fallback libraries.

## HIGH_1 — bind synthetic active universe to frozen selected-500 chain

The synthetic proof must mechanically exercise the same deterministic universe-selection chain as the frozen design.

Required corrections:

1. Use the exact frozen seed text, byte-for-byte:

```text
V13_CONDITIONAL_CROSS_SECTIONAL_SHORT_HORIZON|f9c38ad771710ffd157ac4fad0da15185db82707
```

2. `synthetic_manifest()` must produce the deterministic selected-500 list using production `select_universe(...)`.
3. `synthetic_metadata()` must derive its active synthetic code subset only from `synthetic_manifest()["selected"]`; do not separately invent `1000..1011` or any code outside selected-500.
4. Use a deterministic 12-code active subset from selected-500, split into two synthetic sectors with six members each, preserving Stage-B >=5 coverage.
5. `raw_ohlcv()` must generate price rows only for that active subset.
6. Every active code must be in selected-500 and no excluded code may be active.
7. The complete `run_synthetic_feasibility()` path must consume that same manifest-derived metadata/price chain.

Required regression tests:
- exact seed string;
- selected count exactly 500;
- active codes are a subset of selected-500;
- active codes are disjoint from exclusions;
- raw OHLCV codes equal the active metadata code set;
- selected-500 hash and active-subset hash are deterministic over repeated runs.

No real JPX identity or market data may be introduced; these remain visibly artificial codes/data.

## HIGH_2 — rolling windows use valid ticker observations, not contiguous calendar sessions

The frozen Stage-A contract is explicit: rolling windows use valid ticker observations and may skip invalid/missing ticker observations without forward fill/imputation.

Refactor Stage A so that:

- signal date `t` itself must be a valid ticker observation;
- a valid observation for these synthetic mechanics has all required raw/split-adjusted fields finite and strictly positive where they are divisors/required inputs;
- eligibility requires at least 252 valid observations before `t`, plus valid `t`;
- `ret_k` uses the kth prior valid observation;
- median traded value 20 uses `t` plus previous 19 valid observations;
- Amihud/volatility 20 use 20 `ret_1` observations ending at `t`, therefore enough valid close history to form those 20 returns;
- 52-week high uses `t` plus previous 251 valid observations;
- no condition may require the most recent 20 exchange-calendar sessions to all be valid;
- no forward fill/imputation.

Remove the blocked implementation's contiguous `calendar_window` requirement or any equivalent calendar-contiguity gate.

Required regression tests:
1. Insert one invalid/missing observation inside the latest 20 calendar sessions while retaining >=253 valid observations overall; Stage A must still produce the ticker using the previous valid observation needed to complete the 20-valid-observation window.
2. Independently calculate and assert median traded value, volatility/Amihud window membership, and at least one `ret_k` against the valid-observation sequence.
3. Signal-date invalidity still excludes the ticker.
4. Fewer than 253 valid observations including t excludes the ticker.

## HIGH_3 — criterion J is positive annual mean IC, not >0.01

Frozen criteria are:

```text
I: mean daily IC > 0.01
J: positive mean-IC years >= 4 of 6
```

Correct `_derive_criteria` so J counts years whose defined annual mean IC is strictly `> 0`, not `> 0.01`.

Required regression test: construct diagnostics with four annual mean IC values in `(0, 0.01]`; J must be true when four of six are positive even though those individual values do not exceed 0.01. Separately retain I's global mean-IC `>0.01` threshold.

## MEDIUM_1 — Q4 evidence must be independently derived

Q4 means Stage B reference population is frozen once per date and is not recomputed after Stage-C transformation/omission.

Do not prove Q4 by assigning the same hash to both `reference_hash` and `stage_b_hash` and comparing them.

Implement audit evidence that is mechanically derived from separate points in the production path. At minimum:

- capture the ordered Stage-B member identity tuple/hash immediately after Stage-B population construction and before Stage-C transforms;
- after the complete Stage-C pass, independently derive the ordered identity tuple/hash from the still-frozen Stage-B backing population used for transforms;
- record rank-eligible identities separately;
- Q4 is true only when the pre-Stage-C Stage-B identity and post-Stage-C backing-population identity are equal and no recomputation event occurred;
- Stage-C omission may reduce rank-eligible membership without changing the frozen Stage-B identity.

Expose enough audit fields for behavioral tests; do not merely set a `q4=true` flag.

Required tests:
- pre/post Stage-B identity hashes are produced by separate calculations and match on the success fixture;
- rank-eligible identity is separately reported;
- a synthetic unit-level case that omits a ticker after Stage B does not cause Stage-B reference membership/hash recomputation;
- Q4 in `run_synthetic_feasibility()` is derived from these audit facts.

If producing a post-Stage-B omission fixture requires a small test-only helper/input to `build_rank_population_audit`, keep it deterministic and methodology-neutral. Do not alter Stage-C scientific semantics merely to create a test case.

## Preserve prior successful closures

Do not regress:
- real LightGBM/sklearn/SciPy imports only;
- exact one-pass Stage C over 19 stock-varying fields;
- complete single production orchestrator;
- CASH comparator;
- RANDOM_500 base/stress 500 paths with ranking identity;
- exact four frozen blob bindings;
- no 2026 prices;
- derived A-P/Q1-Q12 mechanics;
- synthetic-only/no research verdict;
- deterministic CLI output.

## Tests/checks

Use the real project interpreter.

Required:
- at least 45 collected targeted pytest cases;
- all targeted tests pass;
- two CLI runs with byte-identical stdout;
- identical SHA256 for both runs;
- exact frozen bindings remain 4/4 PASS;
- source/script static no-network/no-real-data dependency check remains PASS;
- `git diff --check` PASS;
- exactly six allowed files changed;
- no user-specific absolute path newly added.

The executor must inspect the actual final diff and perform a bounded mechanical closure sweep for the four findings before commit.

## PROJECT_STATE

Fold the Issue #32 GPT PASS and current remediation into the same commit. Update existing unique keys, do not append duplicates:

```text
last_gpt_reviewed_sha=e2d3a870428e7706e751915fc46ff468f42996cf
last_gpt_review_result=PASS_CRITICAL_0_HIGH_0_MEDIUM_0_LOW_0
V13_IMPLEMENTATION_REMEDIATION_POLICY_AMENDMENT=GPT_REVIEWED_PASS
V13_IMPLEMENTATION_REMEDIATION_POLICY_AMENDMENT_REVIEWED_SHA=e2d3a870428e7706e751915fc46ff468f42996cf
V13_IMPLEMENTATION_REMEDIATION_POLICY_AMENDMENT_BLOB_SHA1=8121c772ba033f2c6774daabf04fb3f606e7be85
V13_IMPLEMENTATION_REMEDIATION_HARD_CAP=false
V13_SUBSTANTIVE_REMEDIATION_ROUNDS_USED=2
V13_SUBSTANTIVE_REMEDIATION_ROUNDS_REMAINING=NOT_APPLICABLE_NO_HARD_CAP
V13_ACTIVE_IMPLEMENTATION_BUDGET_HARD_STOP=false
V13_ENGINEERING_ESCALATION_THRESHOLD_MINUTES=240
V13_SYNTHETIC_FEASIBILITY_IMPLEMENTATION=METHOD_PRESERVING_REMEDIATION_AWAITING_GPT_REVIEW
V13_SYNTHETIC_FEASIBILITY_HIGH_1=REMEDIATED_AWAITING_GPT_REVIEW
V13_SYNTHETIC_FEASIBILITY_HIGH_2=REMEDIATED_AWAITING_GPT_REVIEW
V13_SYNTHETIC_FEASIBILITY_HIGH_3=REMEDIATED_AWAITING_GPT_REVIEW
V13_SYNTHETIC_FEASIBILITY_MEDIUM_1=REMEDIATED_AWAITING_GPT_REVIEW
current_stage=V13_SYNTHETIC_FEASIBILITY_METHOD_PRESERVING_REMEDIATION_AWAITING_GPT_REVIEW
```

Measure this task's active implementation minutes and add them to the existing cumulative 46 minutes. If cumulative time exceeds 240, set `V13_ENGINEERING_BUDGET_ESCALATION=true`; otherwise false or omit if repository convention prefers absence. Crossing 240 is not a study STOP.

Preserve all real/outcome authority booleans false and future profitability false.

## PROJECT_DECISION_LOG

Append concise chronology:
- Issue #32 exact-SHA PASS at `e2d3a870...`;
- implementation hard-cap removed for methodology-preserving pre-measurement fixes per human-approved policy;
- Issue #31 H1/H2/H3/M1 remediation details;
- tests/CLI hash and cumulative active minutes;
- no irreversible/outcome boundary crossed;
- LOW_1 historical absolute path remains safe-deferred append-only history and is not repeated.

## Prohibitions

- no edit frozen design/amendment/approval artifacts;
- no scientific methodology change;
- no package/environment mutation;
- no network/JPX/Yahoo;
- no real ticker selection;
- no existing real historical OHLCV/cache read;
- no outcome-bearing historical fit/backtest;
- no tuning/grid search;
- no forward/private/sealed/trading action;
- no synthetic profitability or historical-viability claim.

## Commit/push

If all required checks pass and no strict scientific/authority STOP applies:
- create exactly one commit parented to `e2d3a870428e7706e751915fc46ff468f42996cf`;
- immediately before push verify authoritative remote HEAD still equals that parent;
- non-force push to `refs/heads/v13-conditional-cross-sectional-short-horizon`;
- verify remote HEAD and clean working tree;
- do not stop merely because the historical 2-round counter is exhausted; Issue #32 has superseded that hard-stop meaning for this task class.

## Required report

```text
MODEL=Codex
MODE=GITHUB_ISSUE_EXECUTION
TASK=V13_SYNTHETIC_FEASIBILITY_METHOD_PRESERVING_REMEDIATION_3
GITHUB_ISSUE=33
STARTING_HEAD=
CHANGED_FILES=
HIGH_1_STATUS=REMEDIATED_AWAITING_GPT_REVIEW
HIGH_2_STATUS=REMEDIATED_AWAITING_GPT_REVIEW
HIGH_3_STATUS=REMEDIATED_AWAITING_GPT_REVIEW
MEDIUM_1_STATUS=REMEDIATED_AWAITING_GPT_REVIEW
EXACT_FROZEN_SEED_USED=true
ACTIVE_CODES_SUBSET_OF_SELECTED_500=true
ROLLING_WINDOWS_USE_VALID_OBSERVATIONS=true
CRITERION_J_USES_GT_ZERO=true
Q4_INDEPENDENT_STAGE_B_AUDIT=true
REAL_PROJECT_VENV_USED=true
DEPENDENCY_FALLBACKS_PRESENT=false
PACKAGE_INSTALLATIONS=0
ENVIRONMENT_MUTATIONS=0
SYNTHETIC_ONLY_NOT_RESEARCH_EVIDENCE=true
REAL_MARKET_DATA_USED=false
PUBLIC_DATA_NETWORK_REQUESTS=0
REAL_TICKER_SELECTIONS=0
HISTORICAL_PRICE_READS=0
OUTCOME_BEARING_MODEL_FITS=0
HISTORICAL_BACKTESTS=0
TARGETED_TEST_COLLECTED=
TARGETED_TEST_RESULT=
SYNTHETIC_CLI_RUN_1=
SYNTHETIC_CLI_RUN_2=
SYNTHETIC_OUTPUT_SHA256=
DETERMINISM_CHECK=
TASK_ACTIVE_MINUTES=
CUMULATIVE_ACTIVE_IMPLEMENTATION_MINUTES_USED=
ENGINEERING_BUDGET_ESCALATION=
SCIENTIFIC_METHODOLOGY_CHANGED=false
EXECUTION_AUTHORITY_CHANGED=false
FROZEN_DESIGN_CHANGED=false
STATIC_CHECK=
COMMIT_SHA=
PARENT_SHA=
PUSH_RESULT=
REMOTE_HEAD=
WORKING_TREE_CLEAN=
NEXT_ACTION=GPT_EXACT_SHA_INDEPENDENT_REVIEW
```

**Comment by ta1k1-arakawa** [comment]
GPT-5.6 Sol exact-SHA independent review

```text
REVIEWED_SHA=5d1c493d19b183cba56c9a5ead114906a4c99de7
PARENT_SHA=e2d3a870428e7706e751915fc46ff468f42996cf
AUTHORITATIVE_REMOTE_HEAD=5d1c493d19b183cba56c9a5ead114906a4c99de7
COMMIT_COUNT_FROM_PARENT=1
CHANGED_FILE_SCOPE=PASS
TARGETED_TEST_REPORTED=52_PASSED
CLI_DETERMINISM_REPORTED=PASS
FROZEN_BASE_DESIGN_CHANGED=false
FROZEN_DETERMINISM_AMENDMENT_CHANGED=false
BASE_APPROVAL_CHANGED=false
AMENDMENT_APPROVAL_CHANGED=false
CRITICAL=0
HIGH=0
MEDIUM=0
LOW=1
RESULT=PASS
LOW_1=INHERITED_USER_SPECIFIC_ABSOLUTE_PATH_IN_APPEND_ONLY_HISTORY_SAFE_DEFERRED
PUBLIC_ACQUISITION_AUTHORIZED=false
HISTORICAL_VIABILITY_AUTHORIZED=false
FUTURE_PROFITABILITY_ESTABLISHED=false
```

Review notes:
- HIGH_1 is closed: the exact frozen seed is restored; selected-500 is produced through production `select_universe`; the active 12-code synthetic subset is derived only from selected-500; synthetic metadata and raw OHLCV use the same active code set; excluded codes are not active.
- HIGH_2 is closed: Stage A now uses valid ticker observations rather than contiguous exchange-calendar sessions, with >=252 prior valid observations plus t, valid-observation return/traded-value/Amihud/volatility/high windows, and no forward fill/imputation.
- HIGH_3 is closed: criterion I remains global mean IC >0.01 while criterion J now counts years whose annual mean IC is strictly >0.
- MEDIUM_1 is closed: Stage-B identity is captured before Stage C and independently recalculated after the transform pass; rank-eligible identity is separate; production Q4 derives from the pre/post identity equality plus no-recomputation audit facts.
- Prior successful closures remain intact: real LightGBM/sklearn/SciPy only, one-pass Stage C, full synthetic orchestrator, CASH, RANDOM_500 base/stress identity, four frozen bindings, no 2026 price use, A-P/Q mechanics, and synthetic-only/no-research-verdict output.
- The only remaining LOW is inherited append-only historical text containing a user-specific absolute path; this commit did not repeat it. It does not affect authority, provenance, methodology, or safety and is safe to defer.
- No GitHub CI is attached to the commit; the task report states 52 targeted tests passed and two CLI runs were byte-identical with SHA256 `3ab204defdba0e0e2e517f14daf774027503a2bab24ed7055bbc3ca99b8e9306`.

PASS condition is satisfied because CRITICAL/HIGH/MEDIUM are all zero.
